api/DOMImplementation.json

```diff
@@ -0,0 +1,247 @@
+              "version_added": true
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "createHTMLDocument": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/DOMImplementation/createHTMLDocument",
+          "support": {
+            "chrome": {
+              "version_added": true
+            },
+            "edge": {
+              "version_added": true
+            },
+            "firefox": {
+              "version_added": "4"
+            },
+            "ie": {
+              "version_added": "6"
+            },
+            "opera": {
+              "version_added": true
+            },
+            "safari": {
+              "version_added": true
+            },
+            "webview_android": {
+              "version_added": true
+            },
+            "chrome_android": {
+              "version_added": true
+            },
+            "edge_mobile": {
+              "version_added": true
+            },
+            "firefox_android": {
+              "version_added": true
+            },
+            "opera_android": {
+              "version_added": true
+            },
+            "safari_ios": {
+              "version_added": true
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "hasFeature": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/DOMImplementation/hasFeature",
+          "support": {
+            "chrome": {
+              "version_added": true
+            },
+            "edge": {
+              "version_added": true
+            },
+            "firefox": {
+              "version_added": true,
+              "notes": [
+                "Since Firefox 19, <code>hasFeature()</code> mostly returns <code>true</code>."
+              ]
+            },
+            "ie": {
+              "version_added": "6"
+            },
+            "opera": {
+              "version_added": true
+            },
+            "safari": {
+              "version_added": true
+            },
+            "webview_android": {
+              "version_added": true
+            },
+            "chrome_android": {
+              "version_added": true
+            },
+            "edge_mobile": {
+              "version_added": true
+            },
+            "firefox_android": {
+              "version_added": true
+            },
+            "opera_android": {
+              "version_added": true
+            },
+            "safari_ios": {
+              "version_added": true
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
```

Comment:
I think this is deprecated now, at least according to https://dom.spec.whatwg.org/#dom-domimplementation-hasfeature

Reply:
Thanks for the review @ddbeck I've just updated it!